running/generatekube/kubedemo-openshift.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
+
+trap 'exit 0' INT
+
+# Setting up some colors for helping read the demo output.
+# Comment out any of the below to turn off that color.
+bold=$(tput bold)
+blue=$(tput setaf 4)
+reset=$(tput sgr0)
+
+read_color() {
+    read -p "${bold}$1${reset}"
+}
+
+echo_color() {
+    echo "${blue}$1${reset}"
+}
+
+setup () {
+    # Verify podman exists
+    if ! hash podman 2>/dev/null; then
+	echo "Unable to find the podman commmand"
+	exit 1
+    fi
+
+    # Verify oc exists
+    if ! hash oc 2>/dev/null; then
+	echo "Unable to find the oc commmand"
+	exit 1
+    fi
+    sudo systemctl restart crio
+    rc=$?; if [[ $rc != 0 ]]; then
+	echo "Unable to start the crio daemon"
+	exit 1
+    fi
+
+    podman image exists quay.io/sallyom/hello-openshift:test
+    rc=$?; if [[ $rc != 0 ]]; then podman pull quay.io/sallyom/hello-openshift:test; fi
+    podman image exists quay.io/sallyom/hello-openshift:test
+    mkdir -p /tmp/hello-openshift-container
+}
+
+cleanPodmanContainers() {
+    podman stop -a -t 1
```

Comment:
Why wait a second?  -t 0?

Reply:
@rhatdan I took those functions from [Brent's script](https://github.com/containers/Demos/blob/master/running/generatekube/kubedemo.sh#L55-#L118) @baude hope that's ok!  just saw this now.  In a perfect world where I wasn't finishing a demo hours (minutes) before presenting it on stage : ), I would've 
a) given proper credit to Brent in this PR
b) created bash magic so both scripts share the functions rather than duplicating them, I'll work on that and submit a follow-up

Reply:
SGTM